src/api/models/iuf/schemaValidator/schemas/iuf-manifest-schema.yaml

```diff
@@ -396,6 +396,17 @@ properties:
                     type: string
                 md5sum:
                   $ref: "#/$defs/MD5Sum"
+                arch:
+                  description: Target architecture for the recipe.
```

Comment:
What is the behavior if `arch` is not specified? Does IMS default to assuming this is an x86_64 recipe? I don't see that documented in [the IMS OpenAPI spec](https://github.com/Cray-HPE/ims/blob/develop/api/openapi.yaml#L2029).

Reply:
Took a look at the code for recipes in IMS. Looks like if not specified it will default to x86_64 for the recipe.

Reply:
Be default, if not specified IMS will default to `x86`.

Reply:
Is this field optional in IUF? If so, does it default to x86_64?

Reply:
Currently it should be optional and it does infact default to x86_64